ChangeLog

```diff
@@ -1,3 +1,7 @@
+2025-01-18  Ben Bolker  <bolker@pop-os>
```

Comment:
Can you make it a real (as in working) email, pretty please?  (There is an option for that to override I could look up, until then just hand-edit)

Reply:
oops.